versions.tf

```diff
@@ -1,14 +1,14 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 1.11.0"
 
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = ">= 2.0.0"
+      version = ">= 4.25.0"
     }
     random = {
       source  = "hashicorp/random"
-      version = ">= 2.0.0"
+      version = ">= 3.7.1"

```

Comment:
Again, similar to my other two comments, this seems like changing for the sake of change but I don't see a reason to make v3.7.1 a minimum requirement to run this module. Using the current value of >= 2.0.0 will still enable people to use v3.7.1. Please revert this change.